components/x-teaser/Props.d.ts

```diff
@@ -133,6 +133,7 @@ export interface Indicators {
 	accessLevel: 'premium' | 'subscribed' | 'registered' | 'free';
 	isOpinion?: boolean;
 	isColumn?: boolean;
+	isPodcast?: boolean;
```

Comment:
Does this mean the top level `type` property will be `audio` in future rather than `podcast`?

Reply:
yes. type audio, subtype podcast, isPodcast in the teaser indicators.

Reply:
Good to know (though I'm sure you've told me this already!)